guidelines/1.3.3.md

```diff
@@ -2,7 +2,7 @@ This document is in beta. Help us by [reporting issues via Github](https://githu
 
 [Back to the overview page](./../index.html)
 
-# Instructions don't rely on sensory characteristics
+# Instructions should not rely on sensory characteristics
```

Comment:
I agree that "should not" or "must not" is clearer than "don't".

I think that we should use "must not", rather than "should not".

Rationale:
* There's a common pattern in automated accessibility checking tools where
- "Must not" is used for WCAG 2.1 AA violations
- "Should not" is used for failures to follow best practices, that are not covered by WCAG 2.1 AA
* I like that pattern
* I think that we'll be using the same pattern when we start covering more best practices, on top of WCAG 2.1 AA